src/main/java/coverage2/Home.java

```diff
@@ -3,6 +3,7 @@
 public class Home {
     private String name;
     private String address1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "address1" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)